source/change-streams/change-streams.rst

```diff
@@ -205,15 +205,9 @@ Driver API
 
   interface ChangeStream extends Iterable<Document> {
     /**
-     * The resume token (_id) of the document the iterator last returned
+     * The resume token to be used for resuming the iterator
```

Comment:
Hmm, now we use this for both resuming the change stream as part of the "Resume Process" and this is the thing we return in `getResumeToken()`. What do you think about making this more vague, like: "A cached resume token"? We describe how it's used later.